cmd/vpptop_local/main.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019 PANTHEON.tech.
+ * Copyright (c) 2020 Cisco and/or its affiliates.
```

Comment:
Note: copyright is not changed  - the original [main.go](https://github.com/PANTHEONtech/vpptop/pull/12/files#diff-04281bd149c6701ed54f51f7d8f1b303) keeps the PANTHEON.tech header. This one was added.